### PR TITLE
MM-49516: fixes toggle switch alignment

### PR DESCRIPTION
**Pull request by koox00**
#### Summary
We had removed some utility classes from _flex.scss, in result the horizontal alignment of toggle switches in the channel moderation admin console page broke.

This commit brings back `justify-content-between` class to fix the issue.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-49516

#### Screenshots

|  before  |  after  |
|----|----|
|image|image 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/admin_console/team_channel_settings/line_switch.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ type Props = {
 const LineSwitch = (props: Props): JSX.Element => {
     const {title, subTitle, singleLine, toggled, onToggle, children, offText, onText, disabled, last, id} = props;
     return (<div>
-        <div className='line-switch d-flex flex-sm-column flex-md-row align-items-sm-start align-items-center justify-content-md-between'>
+        <div className='line-switch d-flex flex-sm-column flex-md-row align-items-sm-start align-items-center justify-content-between'>
```
Comment (M-ZubairAhmed) [inline components/admin_console/team_channel_settings/line_switch.tsx:25]: i am not sure why we do this approach and not just create a css file for this component.

Comment (koox00) [reply components/admin_console/team_channel_settings/line_switch.tsx:25]: @M-ZubairAhmed I find these helper classes on flexboxes actually helpful, 
I tend to use them in code.
If we are going away from this approach I can create a new class to include all relevant CSS.

Comment (M-ZubairAhmed) [reply components/admin_console/team_channel_settings/line_switch.tsx]: i am not sure whats the preferred approach here is. so i am going to approve it nevertheless. defering to @hmhealey for comments on that.

**Review by M-ZubairAhmed — APPROVED** [review]

File: components/admin_console/team_channel_settings/line_switch.tsx (hunk 1 above)
Comment (koox00) [reply components/admin_console/team_channel_settings/line_switch.tsx]: I've already made the change you proposed BTW :)

**Review by michelengelen — APPROVED** [review]

**Pull request merged**